docs/src/modules/java/pages/quickstart-template.adoc

```diff
@@ -227,6 +230,15 @@ The Maven failsafe plugin runs the integration tests when the `it` profile is en
 mvn verify -Pit
 ----
 
+[.group-scala]
+By default the integration and unit test are both invoked by `sbt test`.
+
+[.group-scala]
+[source,bash]
+----
+sbt test
+----
+
```

Comment:
why don't we use `[.tabset]` here? `[.tabset]` is used just a few lines up for similar sbt and mvn commands

Reply:
I think because the content of the tabs alternate after the `[.tabset]` but I guess I could group them and use only the `[.tabset]`. I'm just don't feel very confident touching anything tab-related as I can't see the result until published.